crates/topos-sequencer-subnet-runtime/tests/subnet_contract.rs

```diff
@@ -918,3 +979,246 @@ async fn test_subnet_send_token_processing(
     context.shutdown().await?;
     Ok(())
 }
+
+/// Test multiple send token events in a block
+/// Test is slow, block time is 12 seconds
```

Comment:
```suggestion
// Test multiple send token events in a block
// Test is slow, block time is 12 seconds
```

(While it's great to have comments describing a test, we do not want `rustdoc` to generate documentation for them)